### ISSUE TITLE
SSS & Element Call: no audio/video/call ringing transmitted

### ISSUE THREAD

**Issue by jacotec** [element-hq/element-x-ios#3275]
### Steps to reproduce

I always use my backup-iPhone (11 Pro Max) for testing and did upgrade it to the IOS18 release candidate.

After switching to IOS18, EX-IOS does not transmit audio or video anymore! On the iPhone I see and hear the counterpart, but the other side does not see my video nor my audio anymore. Toggle video and audio off/on a few times does not help either.

This is regardless which side opens the call. Also the same appears using VP8 and H.264 codec.

As it's just 5 days until the Apple world switches to IOS18 this might have some priority ;-)

#### What did you expect?

Audio and Video transmission should also work on IOS18

#### What happened instead?

After updating from 17.6.1 to IOS 18 RC, the IOS18 device does not transmit audio or video anymore. Receive is fine.

### Your phone model

iPhone 11 Pro Max

### Operating system version

IOS 18 RC

### Application version

1.8.0

### Homeserver

Synapse 1.114.0

### Will you send logs?

Yes

**Comment by jacotec** [comment]
Another (maybe related) issue:

When I start a call from the IOS18 device to another account with an iPhone (still on 17.6.1), there is no CallKit notification! I'm only getting a normal Element notification in this case.

Repeating the test with an Element-Web instance calling with the same test account as the IOS18 phone, CallKit works on the receiver side.

So, this sender issue seems to break the receiving side's CallKit as well.

**Comment by jacotec** [comment]
New information:
Logging out and in does not do anything.

But switching from native sliding sync to proxy sliding sync resolves the issue! Counterside rings and also transmitting video and audio works.

So this issue is related to native sliding sync!

**Comment by stefanceriu** [comment]
@jacotec are you still seeing this problem or has it resolved itself since then?

**Comment by jacotec** [comment]
@stefanceriu Nope, seems to work with current versions. My iPhone shows "native Sliding Sync" and calls are working.
